u8timeseries/timeseries.py

```diff
@@ -141,7 +121,8 @@ def first_value(self) -> float:
         float
             The first value of this series
         """
-        return self._values[0]
+        self._assert_univariate()
+        return self._values[0][0]
```

Comment:
how about defining `first_values()` and `last_values()` counterpart functions?

Reply:
That's definitely a good idea. I will do this as part of step 2, since I have an appropriate test file for new multivariate functions.